src/main/scala/ironoxide/v1/beta/BlindIndexSearch.scala

```diff
@@ -0,0 +1,11 @@
+package ironoxide.v1.beta
+
+import com.ironcorelabs.{sdk => jsdk}
+
+case class BlindIndexSearch(private val javaBlindIndexSearch: jsdk.BlindIndexSearch) {
+  def tokenizeQuery(query: String, partitionId: Option[String]): List[Int] =
+    javaBlindIndexSearch.tokenizeQuery(query, partitionId.orNull).toList
```

Comment:
Since you have to convert to a different datatype, you should just `.toSet` here.

Reply:
Good point, Done